### PR TITLE
Policy

### PR DESCRIPTION
**Pull request by AnshG714**
This PR adds the policy cell at the top of the places list in `PlacesViewController`. 
- Right now, the button opens up in a Safari. We can add an embedded WebView later if needed, but this should be good to merge for now.

image

### CODE REVIEW

File: Campus Density/Assets.xcassets/close.imageset/Contents.json [hunk 1]
```diff
@@ -0,0 +1,21 @@
+{
+  "images" : [
+    {
+      "filename" : "Vector.png",
```
Comment (Coreball) [inline Campus Density/Assets.xcassets/close.imageset/Contents.json]: You can probably remove this now and the image too

File: Campus Density/Controllers/PlacesViewController.swift [hunk 2]
```diff
@@ -31,6 +32,7 @@ class PlacesViewController: UIViewController, UIScrollViewDelegate {
     var collectionView: UICollectionView!
     var loadingBarsView: LoadingBarsView!
     var refreshBarsView: LoadingBarsView!
+    var webView: WKWebView!
```
Comment (Coreball) [inline Campus Density/Controllers/PlacesViewController.swift]: This isn't being used yet is it

**Review by Coreball — APPROVED** [review]
Good job AnshG it looks very pretty

File: Campus Density/Controllers/PlacesViewController.swift (hunk 2 above)
Comment (AnshG714) [reply Campus Density/Controllers/PlacesViewController.swift]: Nope, good point I'll remove it

**Pull request merged**